use-cases/housekeeping.md

```diff
@@ -0,0 +1,19 @@
+## Housekeeping
+
+A housekeeping integration pulls live information about the physical state of a space, allows the housekeeping staff to update it from the housekeeping software and pushes this data back into Mews. The sections below contain guidelines regarding the relevant endpoints used.
+
+### Initial configuration
+
+The integration must first retrieve all the spaces the property has configured in Mews using [Get all spaces](operations/enterprises.md#get-all-spaces). The response contains all relevant information needed to map the space configuration of the property in the external system. 
```

Comment:
Again all links are broken now, add `..` to navigate out of `use-case` directory first.

```suggestion
The integration must first retrieve all the spaces the property has configured in Mews using [Get all spaces](../operations/enterprises.md#get-all-spaces). The response contains all relevant information needed to map the space configuration of the property in the external system. 
```